src/commands/deploy/getOrCreateFunctionApp.ts

```diff
@@ -31,10 +32,16 @@ export async function getOrCreateFunctionApp(context: IFuncDeployContext & Parti
         context.activityTitle = localize('functionAppCreateActivityTitle', 'Create Function App "{0}"', nonNullProp(context, 'newSiteName'))
         await wizard.execute();
 
-        const resolved = new ResolvedFunctionAppResource(context as ISubscriptionContext, nonNullProp(context, 'site'));
+        if (context.dockerfilePath) {
+            const resolved = new ResolvedContainerizedFunctionAppResource(context as ISubscriptionContext, nonNullProp(context, 'site'))
+            node = await ext.rgApi.tree.findTreeItem(resolved.id, context);
```

Comment:
nit: You only need to set node once after the if/else. 

You could also do a ternary for `context.dockerfilePath ? new Resolve...`